### PR TITLE
feat: add container build

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+FROM node:22.8.0-alpine3.20
```
Comment (bastelfreak) [inline Dockerfile:1]: any chance we can add a dedicated user , or do we have to run the commands all as root?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+LABEL org.label-schema.maintainer="Voxpupuli Team <info@voxpupuli.org>" \
```
Comment (bastelfreak) [inline Dockerfile:3]: I don't think that email address exists. We usually use voxpupuli@groups.io ( or mailinglist) or pmc@voxpupuli.org.

File: Dockerfile (hunk 1 above)
Comment (rwaffen) [reply Dockerfile:1]: sure thing. i should right from the start with rootless. thx for the hint.

Comment (rwaffen) [reply Dockerfile:1]: hmpf.... implemented an user and it worked, but now when running the container with a mounted code volume it says it cannot find a `.git` folder. but it's there and the user rights seem to be okay.... unangenehm!

Comment (bastelfreak) [reply Dockerfile:1]: ah I think that's our problem we had in the past, the uid in the container isn't the same as owner of `.git` on the host? :(

Comment (rwaffen) [reply Dockerfile:1]: hmm okay, this were some dangling commits. i cleaned my gitrepo but still not a git repo for semantic-release :(

Comment (rwaffen) [reply Dockerfile:1]: i disabled this for the moment. did not find anything on how to fix this. see #3

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**